package.json

```diff
@@ -80,7 +80,7 @@
     "@semantic-release/npm": "^7.0.5",
     "@storybook/addon-actions": "^6.3.4",
     "@storybook/react": "^6.3.4",
-    "@testing-library/react": "^12.1.4",
+    "@testing-library/react": "^13.0.0-alpha.6",
```

Comment:
Just released alpha.7 so it would be nice to use this repo as a canary ;)

Reply:
Thank you! I've addressed this at https://github.com/reactjs/react-transition-group/pull/810/commits/62a44b0de82714f7526b12d5d953fd06acda0e70